src/ripple/app/main/Application.cpp

```diff
@@ -1085,16 +1085,16 @@ bool ApplicationImp::setup()
     }
 
     {
-        PublicKey validationPub;
-        SecretKey validationPriv;
+        PublicKey valPublic;
+        SecretKey valSecret;
         std::string manifest;
         if (config().exists (SECTION_VALIDATOR_TOKEN))
         {
             if (auto const token = ValidatorToken::make_ValidatorToken (
                 config().section (SECTION_VALIDATOR_TOKEN).lines ()))
             {
-                validationPriv = token->validationPrivate;
-                validationPub = derivePublicKey (KeyType::secp256k1, validationPriv);
+                valSecret = token->validationPrivate;
```

Comment:
Thanks for all of the renames private -> secret.  It looks like I may have missed a few.  How would you feel about renaming the `validatorToken::validationPrivate` field?